index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="KO">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <a href="/"><img src="./image/logo.png" alt="linkbrary logo" width="133px" height="24px"></a>
+        <span class="login-button"><a href="/signin.html">로그인</a></span>
+    </nav>
+    <div class="main">
+        <section class="box">
+            <p class="big-title">
+                <span class="font-gradient">세상의 모든 정보</span>를<br>
+                쉽게 저장하고 관리해 보세요
+            </p>
+            <a href="./signup.html"><span class="add-link">링크 추가하기</span></a>
+            <div class="image-box">
+                <img src="./image/add-link.png" alt="링크의 목록들과 검색기능을 보여주는 이미지" width="100%" height="auto" class="align-center">
+            </div>
+        </section>
+        <section class="justify-center">
```

Comment:
이런식으로 사용하는 걸 유틸리티 클래스라고 하는데요~ 
이렇게 작성해주시려면 아래와 같이 각 속성 하나마다 클래스명이 적용되어야 할 것 같아요~

```suggestion
        <section class="flex gap-100 justify-center">
```

Reply:
혹은 지금과 같이 유지하려면 클래스명만 변경해주시면 좋을 것 같아요!

```suggestion
        <section class="section-container">
```

Reply:
감사합니다 다른부분도 다 수정하고있는데 궁금한 부분에만 댓글 달겠습니다
혹시 두가지 방법 중 앞으로 어떤 방법을 쓰면 더 좋을지 추천해주실수 있으실까요?